src/components/BCheckbox/BCheckbox.vue

```diff
@@ -1,12 +1,12 @@
 <template>
   <!-- Group checkboxes: requires group id and label -->
-  <div v-if="groupId && groupLabel" class="column">
-    <div role="group" :aria-labelledby="groupId" class="flex">
-      <label :id="`group-${groupId}`" :class="{ bold: true, disabled: disabled }">
+  <div v-if="groupId && groupLabel" class="b-checkbox b-flex-column">
+    <div role="group" :aria-labelledby="groupId">
+      <label :id="`group-${groupId}`" :class="{ 'b-checkbox-group-label': true }">
         {{ groupLabel }}<span v-if="required" aria-hidden="true">*</span>
       </label>
-      <div class="flex wrap">
-        <div v-for="(checkbox, index) in checkboxes" :key="index" class="flex baseline">
+      <div class="b-flex b-flex-wrap">
```

Comment:
Was `b-flex` already a class because you added the `b-` to it in the HTML but not in the CSS.

Reply:
Haven't we talked about not doing these type of things in another stories PR? Can we get a concrete stance on this, because I'm thoroughly confused.

Reply:
"Not doing these types of things" - I'm not sure what that refers to here. But yes `b-flex` got created in the previous css change

Reply:
i think he means making random changes outside the scope of the current story.

Reply:
FTR Steven and I talked about this on slack and I am updating the description to include comments about the `flex`->`b-flex` and related changes, which is partly what lead to the confusion.